content/privacy-policy/index.md

```diff
@@ -1,29 +1,95 @@
 ---
 title: "Privacy Policy"
-date: 2024-05-16T16:04:57-04:00
+date: 2024-10-28T00:00:00-04:00
 draft: false
 ---
+
 # Privacy Policy
 
-At null NEU, we are committed to protecting the privacy of our website visitors. This Privacy Policy outlines how we collect, use, disclose, and safeguard your information when you interact with our website. Please read this Privacy Policy carefully to understand how we handle your information.
+At null NEU, the Northeastern University chapter of the null community, we are deeply committed to user privacy and data minimization. This Privacy Policy outlines our strict data handling practices when you interact with our website and organization.
+
+## 1. Technical Infrastructure & Data Collection
+
+### 1.1 Website Architecture
+- Our website is built using Hugo, a static site generator
+- Content is stored in markdown and YAML files
+- Deployment is handled through Cloudflare Pages
+- We intentionally minimize JavaScript usage to enhance privacy and security
+
+### 1.2 Information Collection
+We adhere to data minimization principles:
+- **No Personal Data Collection**: Our static website does not collect or store personal information
+- **No Cookies**: We do not use cookies or tracking mechanisms
+- **RSS Feeds**: We use RSS feeds for updates instead of email subscriptions, ensuring user privacy
+- **Cloudflare Analytics**: Limited anonymous metrics may be collected by Cloudflare Pages for operational purposes
+- **No Newsletter Data**: We do not collect email addresses or maintain subscriber lists
+
+## 2. Communication Channels
+
+When you interact with us through our official channels:
+- RSS Feeds for updates
```

Comment:
Can we add _work in progress_ here?

Reply:
Sure yes! or shall we wait for RSS to be done by @Samyukta-14 ?

Reply:
Let's mark it as WIP, I don't think the issue is being worked on at the moment (#22)